guides/release/upgrading/current-edition/glimmer-components.md

```diff
@@ -433,17 +433,17 @@ After:
 ```js
 import Component from '@glimmer/component';
 
-export default class PersonComponent extends Component {
-  get firstName() {
-    return this.args.firstName || 'Kenneth';
+export default class ImageComponent extends Component {
+  get width() {
+    return this.args.width === undefined ? this.args.width : 0;
```

Comment:
Let's use the nullish coalescing operator? (We'll assume that the developer knows how to set their `configs` file to transpile the operator if necessary.)

```suggestion
    return this.args.width ?? 0;
```